lessons/11-css-specificity/index.html

```diff
@@ -44,13 +56,13 @@ <h1>Home Page</h1>
     </header>
 
     <section>
-      <p>
+      <p id="p-id">
         Lorem ipsum <a href="#">dolor sit amet</a>, consectetur adipisicing
         elit. Veniam corporis aspernatur, obcaecati, voluptate veritatis ad
         fugiat voluptatum repellat deserunt dolor magnam sapiente expedita
         harum! Dolorem animi sit fuga. Asperiores, delectus!
       </p>
-      <p>
+      <p id="p-id">
```

Comment:
Close! ID's are unique in HTML, so two elements can't have the same one. 

A class would be a good use here, or targeting the pargraphs inside the section specifically

```css
section p {
  ...
} 
```